Jenkinsfile

```diff
@@ -184,23 +184,19 @@ def PublishNPM() {
 }
 
 def PublishDocs() {
-    if (env.GIT_BRANCH.endsWith(params.MAIN_BRANCH)) {
```

Comment:
@WesleyMayle looks like these changes might break the staging build

Reply:
Sory, I thought I had already grabbed the latest file from develop. Should be fixed now.